README.md

```diff
@@ -28,11 +28,11 @@ To help users to setup the getting started, a setup script is located [here](scr
 
 ### Features and requirements
 
-The `-e` option is used to enable features, for example `-e extras` enable TDP extras. Each feature have requirements, so if you enable a feature, you **MUST** install the feature requirements **BEFORE** launching the setup script.
+The `-e` option is used to enable features, for example, `-e extras` enables [TDP Collection Extras](https://github.com/TOSIT-IO/tdp-collection-extras). Each feature have requirements, so if you enable a feature, you **MUST** install the feature requirements **BEFORE** launching the setup script.
 
 Common requirements are:
 
-- Python = 3.6 with virtual env package (i.e. `python3-venv`)
+- Python >= 3.6 with virtual env package (i.e. `python3-venv`)
```

Comment:
#300 Change to `=`, why to you revert it ?

Reply:
it wasn't made by purpose, fixing